apps/judicial-system/web/src/routes/PublicProsecutor/Indictments/Overview/Overview.tsx

```diff
@@ -121,67 +107,66 @@ export const Overview = () => {
         <Box component="section" marginBottom={5}>
           <IndictmentCaseFilesList workingCase={workingCase} />
         </Box>
-        {!workingCase.indictmentReviewDecision && (
-          <Box marginBottom={5}>
-            <SectionHeading
-              title={fm(strings.reviewerTitle)}
-              description={
-                <Text variant="eyebrow">
-                  {fm(strings.reviewerSubtitle, {
-                    isFine:
-                      workingCase.indictmentRulingDecision ===
-                      CaseIndictmentRulingDecision.FINE,
-                    indictmentAppealDeadline: formatDate(
-                      workingCase.indictmentAppealDeadline,
-                    ),
-                    appealDeadlineIsInThePast:
-                      workingCase.indictmentVerdictAppealDeadlineExpired,
-                  })}
-                </Text>
-              }
-            />
-            <BlueBox>
-              <Select
-                name="reviewer"
-                label={fm(strings.reviewerLabel)}
-                placeholder={fm(strings.reviewerPlaceholder)}
-                value={
-                  selectedIndictmentReviewer
-                    ? selectedIndictmentReviewer
-                    : workingCase.indictmentReviewer
-                    ? {
-                        label: workingCase.indictmentReviewer.name || '',
-                        value: workingCase.indictmentReviewer.id,
-                      }
-                    : undefined
-                }
-                options={publicProsecutors}
-                onChange={(value) => {
-                  setSelectedIndictmentReviewer(value as Option<string>)
-                }}
-                isDisabled={loading}
-                required
-              />
-            </BlueBox>
-          </Box>
+        {!workingCase.indictmentReviewDecision ? (
+          <IndictmentReviewerSelector
+            workingCase={workingCase}
+            selectedIndictmentReviewer={selectedIndictmentReviewer}
+            setSelectedIndictmentReviewer={setSelectedIndictmentReviewer}
+          />
+        ) : (
+          <ReviewDecision
+            caseId={workingCase.id}
+            currentDecision={workingCase.indictmentReviewDecision}
+            indictmentAppealDeadline={
+              workingCase.indictmentAppealDeadline ?? ''
+            }
+            indictmentAppealDeadlineIsInThePast={
+              workingCase.indictmentVerdictAppealDeadlineExpired ?? false
+            }
+            modalVisible={confirmationModal}
+            setModalVisible={setConfirmationModal}
+            isFine={
+              workingCase.indictmentRulingDecision ===
+              CaseIndictmentRulingDecision.FINE
+            }
+            onSelect={(decision?: IndictmentCaseReviewDecision) => {
```

Comment:
Super nit: I tend to think that functions that are longer than one-liners should not be defined inline in the props list but rather above the return statement. Also, there is a general convention in the code to always have parentheses in if statements, regardless of wether it's just to return, i.e.

  ```
  if (!decision) {
      return
  }
  ```